core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -582,17 +582,24 @@ public static PrintStream getOut() {
         List<URL> urls() {
             List<URL> urls = new ArrayList<>();
             if (version != null) {
-                // Elasticsearch download service
-                addUrl(urls, "http://download.elasticsearch.org/" + user + "/" + repo + "/" + repo + "-" + version + ".zip");
-                // Maven central repository
-                addUrl(urls, "http://search.maven.org/remotecontent?filepath=" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
-                // Sonatype repository
-                addUrl(urls, "https://oss.sonatype.org/service/local/repositories/releases/content/" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
-                // Github repository
-                addUrl(urls, "https://github.com/" + user + "/" + repo + "/archive/" + version + ".zip");
+                // Elasticsearch new download service uses groupId org.elasticsearch.plugins from 2.0.0
+                if (user == null) {
+                    addUrl(urls, "http://download.elasticsearch.org/org.elasticsearch.plugins/" + repo + "/" + repo + "-" + version + ".zip");
+                } else {
+                    // Elasticsearch old download service
+                    addUrl(urls, "http://download.elasticsearch.org/" + user + "/" + repo + "/" + repo + "-" + version + ".zip");
+                    // Maven central repository
+                    addUrl(urls, "http://search.maven.org/remotecontent?filepath=" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
+                    // Sonatype repository
+                    addUrl(urls, "https://oss.sonatype.org/service/local/repositories/releases/content/" + user.replace('.', '/') + "/" + repo + "/" + version + "/" + repo + "-" + version + ".zip");
+                    // Github repository
+                    addUrl(urls, "https://github.com/" + user + "/" + repo + "/archive/" + version + ".zip");
```

Comment:
not really scope of this PR, but otherwise I will do it.. if we use `String.format()` for those URLs, they will be more readable